spec/controllers/selects_collections_controller_spec.rb

```diff
@@ -9,8 +9,9 @@ class SelectsCollectionsController < ApplicationController
 describe SelectsCollectionsController, type: :controller do
   describe "#find_collections" do
     it "uses the search builder" do
-      expect(subject.collections_search_builder_class.default_processor_chain).to eq [:default_solr_parameters, :add_query_to_solr, :add_access_controls_to_solr_params, :add_collection_filter, :some_rows, :sort_by_title]
-      expect(CurationConcerns::CollectionSearchBuilder).to receive(:new).with(subject).and_call_original
+      expect(subject.collections_search_builder_class.default_processor_chain)
+        .to include(:default_solr_parameters, :add_query_to_solr, :add_access_controls_to_solr_params)
```

Comment:
I don't check the remaining methods in the other test, so I left them in here.  I'm amenable to an argument for removal, but since SelectsController is deprecated anyway, I'd just as soon not spend additional time on its tests.